phoenix-core/src/main/java/org/apache/phoenix/jdbc/PhoenixEmbeddedDriver.java

```diff
@@ -481,16 +514,20 @@ private Configuration getConfiguration(ReadOnlyProps props, Properties info, Str
             }
             return config;
         }
-        
+
         private final Integer port;
         private final String rootNode;
         private final String zookeeperQuorum;
         private final boolean isConnectionless;
         private final String principal;
         private final String keytab;
         private final User user;
-        
+
         public ConnectionInfo(String zookeeperQuorum, Integer port, String rootNode, String principal, String keytab) {
+            this(zookeeperQuorum, port, rootNode, principal, keytab, null);
+        }
+
+        public ConnectionInfo(String zookeeperQuorum, Integer port, String rootNode, String principal, String keytab, String bootstrap) {
```

Comment:
Can we consider a parsed Enum or was there a reason you used a String for bootstrap?

Reply:
Been writing a lot of Python lately. _whats an enum_?